roofit/hs3/inc/RYMLParser.h

```diff
@@ -0,0 +1,68 @@
+#ifndef RYML_PARSER_H
+#define RYML_PARSER_H
+#include "JSONInterface.h"
+
+#ifdef INCLUDE_RYML
```

Comment:
In ROOT we usually use `R__HAS_XXX` macro defined in pregenerated `RConfigure.h`

For this, you need to add here relevant lines for `ryml` https://github.com/root-project/root/blob/master/cmake/modules/RootConfiguration.cmake#L504-L508 (use as an example `liburing`) and check https://github.com/root-project/root/blob/master/config/RConfigure.in#L71

it will mean that you will not need `SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -DINCLUDE_RYML")` in your CMakeLists.txt, but just to add `#include "RConfigure.h"` where you use `#ifdef INCLUDE_RYML`